### PR TITLE
docs: update search assets to work on dark background

### LINKED ISSUE #14365: Update search docs to render nicely on dark mode
- [ ] We use some icons that are not dark mode friendly. I'm not sure how to set the image based on theme, will figure it out.

On page: https://docs.sourcegraph.com/user/search/queries

Screen Shot 2020-10-01 at 9 46 51 PM

- [ ] language reference page renders white background for diagrams always:

Screen Shot 2020-10-01 at 9 47 53 PM

- [ ] language reference page should not change code style since the contrast issue that bothered me before is now gone:

Screen Shot 2020-10-01 at 9 48 15 PM

### PR DESCRIPTION
**Pull request by rvantonder**
Closes #14365.

Renders correctly now:

Screen Shot 2020-10-20 at 6 48 18 PM

Screen Shot 2020-10-20 at 6 55 01 PM

### CODE REVIEW

File: doc/code_search/reference/language.md [hunk 1]
```diff
@@ -848,6 +829,7 @@ conflict with the syntax of parameters in this Sourcegraph language.
                 </td>
                 <td class="d"><code class="c"><a href="#commit-parameter">commit parameter</a></code></td>
                 <td class="le"></td>
+                </td>
```
Comment (christinaforney) [inline doc/code_search/reference/language.md:832]: I think this is an extra closing `</td>`

**Review by christinaforney — CHANGES_REQUESTED** [review]
This is awesome! Unfortunately it looks like light mode is broken now: 
image

**Comment by sqs** [comment]
You could try using CSS `filter: brightness(N%)` and maybe `hue-rotate(180deg)` too to make a white-foreground PNG dark in the light theme, without needing to upload duplicate images.

**Comment by rvantonder** [comment]
Ah, thanks for catching @christinaforney, I assumed (wrongly) the docs are only in dark mode now since we only had one theme before. I'll figure out a solution to make this work for both. 

FWIW I tried `invert(100%)` before changing the assets but that would only invert the non-transparent part of the PNGs and not apply to the borders. I'll play around.

**Comment by rvantonder** [comment]
@christinaforney I fixed things to work in dark and light mode. 

Got `invert(100%)` working for the toggle PNGs so I don't need extra dark versions, but doing the same for the railroad assets  is just a big pain and much easier to just override.

**Review by stefanhengl — APPROVED** [review]

**Comment by rvantonder** [comment]
@christinaforney feel free to leave post merge feedback if any

**Comment by christinaforney** [comment]
Looks good!

**Pull request merged**